.github/workflows/release.yml

```diff
@@ -35,7 +35,28 @@ jobs:
         uses: pnpm/action-setup@v3
         with:
           version: 8
-          run_install: true
+          run_install: false
+
+      - name: Get pnpm store directory
```

Comment:
is this step needed?

Reply:
Yes, I believe so. It sets the `STORE_PATH` variable, which is used in the subsequent `actions/cache` step (`steps.pnpm-cache.outputs.STORE_PATH`). This configures the caching path for installed pnpm dependencies.

Reply:
ah, I see now